ui-admin/src/study/kits/KitEnrolleeSelection.tsx

```diff
@@ -69,9 +69,12 @@ export default function KitEnrolleeSelection({ studyEnvContext }: { studyEnvCont
     setEnrollees(enrolleeRows)
   }, [studyEnvContext.study.shortcode, studyEnvContext.currentEnv.environmentName])
 
-  const onSubmit = async () => {
+  const onSubmit = async (success: boolean) => {
     setShowRequestKitModal(false)
     reload()
+    if (success) {
```

Comment:
```suggestion
   /**  if any kits were created, that changes the filter state of the table
       and could result in hidden items still being selected.  Clear the selections to be safe */
```

Reply:
Did you mean to suggest removing the if condition?